dev/Repeater/APITests/RepeaterTests.cs

```diff
@@ -148,6 +149,27 @@ public void ValidateGetSetItemsSource()
             });
         }
 
+        [TestMethod]
+        public void ValidateNullItemsSource()
+        {
+            RunOnUIThread.Execute(() =>
+            {
+                int hResult = 0;
+                ItemsRepeater repeater = new ItemsRepeater();
+                try
+                {
+                    repeater.GetOrCreateElement(0);
+                }
+                catch (COMException e)
+                {
+                    hResult = e.HResult;
+                }
+                //Make sure that we threw E_FAIL
+                Verify.AreEqual((uint)hResult, 0x80004005);
```

Comment:
I think it would be better to check against the message, not the hresult as this number has little meaning for developers; the exception message what will mainly point devs to the correct direction.